documentation/modules/auxiliary/scanner/http/jira_user_enum.md

```diff
@@ -0,0 +1,59 @@
+## Description
+
+  This module exploits an information disclosure vulnerability that allows an unauthenticated user to enumerate users in the /ViewUserHover.jspa endpoint.
+  This only affects Jira versions < 7.13.16, 8.0.0 ≤ version < 8.5.7, 8.6.0 ≤ version < 8.12.0
```

Comment:
I verified and it works fine with Jira version 8.4.1. However, I tested against Jira version 8.11.1, which should be an affected version, and it doesn't seems to work:

![Screen Shot 2021-02-12 at 17 46 52](https://user-images.githubusercontent.com/56716719/107796634-60982500-6d5a-11eb-9be0-1f9170d7db5d.png)
```
msf6 auxiliary(scanner/http/jira_user_enum) > options

Module options (auxiliary/scanner/http/jira_user_enum):

   Name              Current Setting  Required  Description
   ----              ---------------  --------  -----------
   BRUTEFORCE_SPEED  5                yes       How fast to bruteforce, from 0 to 5
   Proxies                            no        A proxy chain of format type:host:port[,type:host:port][...]
   RHOSTS            127.0.0.1        yes       The target host(s), range CIDR identifier, or hosts file with syntax 'file:<path>'
   RPORT             8081             yes       The target port (TCP)
   SSL               false            no        Negotiate SSL/TLS for outgoing connections
   TARGETURI         /                yes       Jira Path
   THREADS           1                yes       The number of concurrent threads (max one per host)
   USERNAME          test1            no        A specific username to authenticate as
   USER_FILE                          no        File containing usernames, one per line
   VERBOSE           true             yes       Whether to print output for all attempts
   VHOST                              no        HTTP server virtual host

msf6 auxiliary(scanner/http/jira_user_enum) > run

[*] Begin enumerating user at 127.0.0.1/secure/ViewUserHover.jspa?username=
[*] checking user test1
[-] No Response From Server
[-] 127.0.0.1:8081 - [1/1] - Bruteforce cancelled against this service.
[*] http://127.0.0.1:8081/ - No users found.
[*] Scanned 1 of 1 hosts (100% complete)
[*] Auxiliary module execution completed
```